doc/Makefile.am

```diff
@@ -194,22 +194,29 @@ geany-sciwrappers-gtkdoc-tmp.h: geany-sciwrappers-gtkdoc.h
 		-e 's,sci_,scintilla_object__GI__MARK_,g' \
 		$< > $@
 
+geany-gtkdoc-tmp.h: geany-gtkdoc.h
+	$(AM_V_GEN) $(SED) \
+		-e 's,\(document_\)\(new_file\),\1_GI__WORKAROUNDNEW\2,g' \
```

Comment:
Yeah I wasn't 100% sure but I felt the GIR devs couldn't be that careless, so I went ahead and tried, and it worked fine, so I chose that one because it's more obvious and easier to strip.